src/uct/ib/rdmacm/ucx-rdmacm.pc.in

```diff
@@ -0,0 +1,16 @@
+#
+# Copyright (c) 2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# See file LICENSE for terms.
+#
+
+prefix = @prefix@
+exec_prefix = @exec_prefix@
+libdir = @libdir@/ucx
+
+Name: @PACKAGE@-rdmacm
+Description: Unified Communication X Library RDMACM module
+Version: @VERSION@
+Libs:
+Libs.private: -L${libdir} -luct_rdmacm -Wl,--undefined=uct_rdmacm_init @IBVERBS_LDFLAGS@ @RDMACM_LDFLAGS@ @RDMACM_LIBS@
+Requires.private: libibverbs
```

Comment:
should be librdmacm (and can probaly remove it from Lib.sprivate)
```
$ pkg-config --libs --static librdmacm
-L/usr//usr/lib64 -lrdmacm -lmlx5 -libverbs -lpthread -lnl-route-3 -lnl-3
```

Reply:
> should be librdmacm (and can probaly remove it from Lib.sprivate)

ok, updated